RELEASE.md

```diff
@@ -1,30 +1,83 @@
 # How to release Kuadrant Operator
 
-## Process
+To release a version _“vX.Y.Z”_ of Kuadrant Operator in GitHub and Quay.io, there are two options, a manual and an
+automated process. For both processes, first make sure every [Kuadrant Operator dependency](https://github.com/Kuadrant/kuadrant-operator/blob/main/RELEASE.md#kuadrant-operator-dependencies)
+has been already released.
```

Comment:
```suggestion
To release a version _“vX.Y.Z”_ of Kuadrant Operator in GitHub and Quay.io, there are two options, a manual and an automated process.
For both processes, first make sure every [Kuadrant Operator dependency](https://github.com/Kuadrant/kuadrant-operator/blob/main/RELEASE.md#kuadrant-operator-dependencies)
has been already released.
```

The formatting is a bit odd here. Markdown supports new line for sentences. The new lines will be rendered as a paragraph. But this had some strange splitting.

Reply:
Thanks! seems like a typo

Reply:
Sorry, I was confused by the line markers, thanks again!